src/java.base/share/classes/java/util/concurrent/ConcurrentHashMap.java

```diff
@@ -848,7 +848,7 @@ public ConcurrentHashMap(int initialCapacity) {
      * @param m the map
      */
     public ConcurrentHashMap(Map<? extends K, ? extends V> m) {
-        this.sizeCtl = DEFAULT_CAPACITY;
+        this(m.size(), LOAD_FACTOR, 1);
```

Comment:
This looks like just `this(m.size())`, right? Not sure if we want to save an additional chained constructor call, though.